Codex follow-up review:

## Finding 1 [P1]
File: `app/routes/login.test.tsx`, `app/routes/register.test.tsx`  
Lines: imports around `15-16` in both new test files  
Problem: the follow-up fixed `__dirname`, but the new tests still live under `app/routes/` and now import Node built-ins (`fs`, `url`). `tsconfig.cloudflare.json` includes `app/**/*` while restricting ambient types to `["vite/client"]`, so these files are part of the Cloudflare typecheck program without Node module declarations. I reproduced the same import pattern in a `/tmp` copy and `npx tsc -p tsconfig.cloudflare.json --noEmit` reports `TS2307: Cannot find module 'fs'` and `TS2307: Cannot find module 'url'`.  
Impact: the required unit regression coverage for the Apple SSO removal can still break the project typecheck/CI path even though the Vitest runtime path is now ESM-safe. This blocks the PR from being merge-ready.  
Required fix: keep Node-based static source tests out of the Cloudflare app compile. The lowest-risk fix is to exclude test files from `tsconfig.cloudflare.json`, e.g. add `exclude: ["app/**/*.test.ts", "app/**/*.test.tsx"]`, or move these static tests to a test-only location/config that is not included by the Cloudflare project reference. Do not add Node types globally to the Cloudflare app config unless the repo explicitly wants Node ambient types in Worker code.  
Verification: rerun `npx tsc -p tsconfig.cloudflare.json --noEmit` and `npm run test -- app/routes/login.test.tsx app/routes/register.test.tsx`; both should pass with the Apple removal tests still active.

Context checked:
- PR body: checked via GitHub PR metadata for #185.
- Linked issue/story: tadaify-app#183 scope checked from PR body and test references; related tadaify-app#131 closure noted from PR body.
- BR: `BR-AUTH-05` checked in `docs/BUSINESS_REQUIREMENTS.md`; no BR change needed for this cleanup.
- TR: `docs/TECHNICAL_REQUIREMENTS.md` checked; `TR-tadaify-001` unit-test CI gate is relevant, and no new TR is introduced.
- DEC: `DEC-346=C`, `docs/decisions/0048-apple-sso-dropped-entirely-dec346.md`, and `docs/decisions/INDEX.md` checked in the supplied diff; prior DEC id collision was addressed by sibling PR renumbering per Claude comment.
- Tests: checked `app/routes/login.test.tsx`, `app/routes/register.test.tsx`, and `e2e/sso-providers-list.spec.ts`; the prior `__dirname` issue is fixed with `fileURLToPath(new URL(..., import.meta.url))`.
- Prior review-loop comments checked: prior Codex P1 duplicate DEC id finding, Claude resolution comment, prior Codex P2 `__dirname` finding, and latest clean follow-up comment were checked.
- Tests run by reviewer: `npx tsc -p tsconfig.cloudflare.json --noEmit` against a `/tmp` copy with the same new Node import pattern: fail, 10s, reproduces `TS2307` for `fs` and `url`. Actual PR checkout/test execution could not run because local `git fetch` was blocked by sandbox write restrictions to `.git/FETCH_HEAD`, and network clone from GitHub failed DNS resolution in the shell environment.

Database review:
- No DB, SQL, migration, RLS, RPC, Supabase Edge Function, or schema changes are present in this PR.

Cost impact:
- No meaningful cost impact identified.

Model routing:
- Tier: premium
- Selected model: gpt-5.5
- Rationale: Premium because this PR is high-risk or already in a repeated review/fix loop.

<!-- review-loop: speaker=codex; kind=follow-up; head=4acd340827d8f0b5ac5407a2cc72e70b0f4da2f1; state=findings; version=1 -->

--- a/tsconfig.cloudflare.json
+++ b/tsconfig.cloudflare.json
@@ -8,6 +8,10 @@
     "workers/**/*",
     "worker-configuration.d.ts"
   ],
+  "exclude": [
+    "app/**/*.test.ts",
+    "app/**/*.test.tsx"
+  ],
   "compilerOptions": {
     "composite": true,
     "strict": true,

